### ISSUE TITLE
would be nice to have also the Prayer Time displayed and not only after X hours.

### ISSUE THREAD

**Issue by Emad2504** [home-assistant/home-assistant.github.io#25162]
### Feedback

would be nice to have also the Prayer Time displayed and not only after X hours.

### URL

https://www.home-assistant.io/integrations/islamic_prayer_times#configuration

**Comment by frenck** [comment]
Thank you for reaching out. We use GitHub for tracking issues, not for providing support or tracking feature requests.

If you want to suggest a feature, you should try our [Community Forum: Feature Requests](https://community.home-assistant.io/c/feature-requests).

If you have additional questions, feel free to join our [Discord chat server](https://discord.gg/c5DvZ4e).

Thanks! 👍
